packages/Tabs/README.md

```diff
@@ -0,0 +1,70 @@
+## Tabs
+
+The `<Tabs>` component displays tabs horizontally with panels associated with each tab.
+
+### Installation
+
+`> npm install --save @paprika/tabs`
+or
+`> yarn add @paprika/tabs`
+
+### Usage
+
+```js
+import Tabs from "@paprika/tabs";
+
+<Tabs>
+  <Tabs.List>
+    <Tabs.Tab label="Hello" />
```

Comment:
@oscarkwan I believe this should be on the children prop and no as label prop, if we use it as a children is clear that you can pass whatever you want if you use it as a label gives the impression that only received a string, 

Also in need of passing a complex component over the label prop will be end looking strange, for some developer:

`<Tabs.Tab label={<span><Icon /><Text /></span>} />`

Reply:
Done, thanks!